airgap/uds-dev-stack/zarf.yaml

```diff
@@ -0,0 +1,85 @@
+kind: ZarfPackageConfig
+metadata:
+  name: uds-dev-stack-airgap-images
+  yolo: true
+  description: >
+    *** REQUIRES DOCKER ***
+    Loads the required images for UDS Dev Stack into Docker to be used offline.
+
+constants:
+  - name: LOCAL_PATH_PROVISIONER_VERSION
+    value: "v0.0.31"
+  - name: BUSYBOX_VERSION
+    value: "latest"
+  - name: WOLFI_BASE_VERSION
+    value: "latest"
+  - name: PAUSE_VERSION
+    value: "3.10"
+  - name: NGINX_VERSION
+    value: "1.28.0-alpine3.21"
+  - name: METALLB_CONTROLLER_VERSION
+    value: "v0.14.9"
+  - name: METALLB_SPEAKER_VERSION
+    value: "v0.14.9"
+  - name: FRR_VERSION
+    value: "9.1.0"
+  - name: MINIO_VERSION
+    value: "RELEASE.2025-04-03T14-56-28Z"
+  - name: MINIO_MC_VERSION
+    value: "RELEASE.2024-11-21T17-21-54Z"
+
+components:
+  - name: uds-dev-stack-airgap-images
+    files:
+      - source: local-path-provisioner.tar
+        target: local-path-provisioner.tar
+      - source: busybox.tar
+        target: busybox.tar
+      - source: wolfi-base.tar
+        target: wolfi-base.tar
+      - source: pause.tar
+        target: pause.tar
+      - source: nginx.tar
+        target: nginx.tar
+      - source: metallb-controller.tar
+        target: metallb-controller.tar
+      - source: metallb-speaker.tar
+        target: metallb-speaker.tar
+      - source: frr.tar
+        target: frr.tar
+      - source: minio.tar
+        target: minio.tar
+      - source: minio-mc.tar
+        target: minio-mc.tar
+    actions:
+      onCreate:
+        before:
+          - cmd: |
+              ./zarf tools registry pull --platform linux/"###ZARF_PKG_ARCH###" rancher/local-path-provisioner:${LOCAL_PATH_PROVISIONER_VERSION} local-path-provisioner.tar
```

Comment:
```suggestion
              ./zarf tools registry pull --platform linux/"###ZARF_PKG_ARCH###" rancher/local-path-provisioner:${ZARF_CONST_LOCAL_PATH_PROVISIONER_VERSION} local-path-provisioner.tar
```

Reply:
If you want to do constants - it does add complexity that might not be needed?

Reply:
No more constants